src/shared/api.ts

```diff
@@ -2403,6 +2407,95 @@ export const cerebrasModels = {
 	},
 } as const satisfies Record<string, ModelInfo>
 
+// Groq
+// https://console.groq.com/docs/models
+// https://groq.com/pricing/
+export type GroqModelId = keyof typeof groqModels
+export const groqDefaultModelId: GroqModelId = "llama-3.3-70b-versatile"
+export const groqModels = {
+	// Compound Beta Models - Hybrid architectures optimized for tool use
+	"compound-beta": {
```

Comment:
You can verify this by going [here](https://console.groq.com/docs/models) then clicking details for every model to check the pricing.